psi-ms.obo

```diff
@@ -22695,6 +22695,81 @@ name: environment metric
 def: "QC metric related to measurements of the ambient environment, such as the laboratory." [PSI:MS]
 is_a: MS:4000001 ! QC metric
 
+
+[Term]
+id: MS:4000025
```

Comment:
Discussion on whether this should be a table [here](https://github.com/HUPO-PSI/mzQC/issues/244#issuecomment-2365126361).

The metric type is fully missing in any case, so no indication whether this is a table or n-tuple currently.

Reply:
I've specified it as an n-tuple now, simply because there information on spectrum IDs may not be available or required.
If needed, we can always add another term for a table, which uses `MS:4000025` as a column. We'd need the column definition as a separate term anyway.